src/lib/loaders/loaders_without_authentication/gravity.ts

```diff
@@ -211,6 +211,11 @@ export default (opts) => {
       (id) => `sale/${id}/sale_artworks`,
       { headers: true }
     ),
+    saleArtworksAllLoader: gravityLoader(
+      "sale_artworks",
+      {},
+      { headers: true }
+    ),
```

Comment:
Adding the loader here to be able to use it without a user token. The Gravity endpoint ensures that passing a user ID as a param is only allowed for trusted clients.

Reply:
As long as necessary checks in gravity are in place 👍 . Wouldn't this allow anyone with a user's id obtain info about who a user follows based on the artworks artists?

Reply:
This line here should make sure that only trusted clients can obtain information about other user's follows: https://github.com/artsy/gravity/pull/16258/files#diff-4b1f68de7f2011be67da60995512be060d31e95c4ebe59ee50df7728efb5db15R32-R33